resources/views/dashboard.blade.php

```diff
@@ -14,7 +14,7 @@
                 </div>
             </x-invitation>
         @endif
-        <p>{{ $user->name }}@if($memberable), {{ $memberable->name }}@endif</p>
```

Comment:
The `@if` tag here no longer does anything— it needs to be wrapped around the `{{ $memberable->name }}` (this accounts for one of the test failures as there are some instances where a user is not a member of an organization).